test/e2e/data/infrastructure-vsphere-govmomi/main/base/patch-k8s-install-script-kcp.yaml

```diff
@@ -55,7 +65,7 @@
         CONTAINER_EXT="tar"
         echo "* testing CI version $${KUBERNETES_VERSION}"
         # Check for semver
-        if [[ "$${KUBERNETES_VERSION}" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
+        if [[ "$${DISTRO}" == "ubuntu" ]] && [[ "$${KUBERNETES_VERSION}" =~ ^v[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
Q: Wasn't the plan to drop this entire branch?

Reply:
Ah yes, forgot about that 😂